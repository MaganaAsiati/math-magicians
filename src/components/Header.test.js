import React from 'react';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from './Header';

const MockHeader = () => (
  <BrowserRouter>
    <Header />
  </BrowserRouter>
);

test('renders correctly', () => {
  const tree = renderer
    .create(<MockHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});