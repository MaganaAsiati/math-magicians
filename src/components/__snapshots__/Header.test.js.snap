// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <h1
    className="h1"
  >
    Math Magicians
  </h1>
  <section
    className="navLinksSection"
  >
    <a
      aria-current="page"
      className="navLink active"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <h2>
      |
    </h2>
    <a
      className="navLink"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <h2>
      |
    </h2>
    <a
      className="navLink"
      href="/Quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </section>
</div>
`;
